http-builder-ng-core/src/test/groovy/groovyx/net/http/JavaHttpPatchSpec.groovy

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2017 Keith Suderman
```

Comment:
This should be changed to "HttpBuilder-NG Project" - as all of the others will be shortly.